.buildkite/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+---
+steps:
+  - label: ":hammer: Test"
+    agents: queue=${AGENT_QUEUE}
+    plugins:
+      - docker-compose#v3.1.0:
+          run: tests
+
+  - label: ":sparkles: Lint"
+    agents: queue=${AGENT_QUEUE}
+    plugins:
+      - plugin-linter#v2.0.0:
+          id: envato/heroku-container-deploy
+
+  - label: ":shell: Shellcheck"
+    agents: queue=${AGENT_QUEUE}
+    plugins:
+      - shellcheck#v1.1.2:
+          files:
+            - hooks/**
```

Comment:
How are you planning to run these tests? Be aware we do not allow public repos to run tests on our internal CI servers.

Reply:
I have in the past used github actions https://github.com/envato/docker-cache-buildkite-plugin

Reply:
Happy to take advice on this, however I have replicated https://github.com/envato/aws-s3-sync-buildkite-plugin/tree/main/.buildkite and verified that third party fork builds are disabled.

Reply:
Github actions would be better because it would allow all developers to view CI builds/failures, not just developers at Envato

Reply:
Solid point. Updated